launch_ros/launch_ros/actions/node.py

```diff
@@ -196,6 +196,7 @@ def __init__(
         :param: remappings ordered list of 'to' and 'from' string pairs to be
             passed to the node as ROS remapping rules
         :param: ros_arguments list of ROS arguments for the node
+        :param: prefix command placed in front of executable on launch
```

Comment:
We talked about this in the issue triage meeting, and the conclusion was that we should reference the base class that actually handles this argument, e.g. have a look at how Python does this for the subprocess API: https://docs.python.org/3/library/subprocess.html#subprocess.run

It's fine to give a brief description of the argument, but it should be separate from the other (actual) arguments and it should reference the place where it's actually documented, e.g. here: https://github.com/ros2/launch/blob/651f57451e4d0e93d055669764fcd588e2d9b1e6/launch/launch/actions/execute_process.py#L211-L216

It is indeed helpful to connect the dots here, but we don't want to duplicate too much documentation and risk them getting out of sync.

Reply:
Okay that makes total sense. Thank you for your reply